server/config/constants.go

```diff
@@ -31,8 +31,9 @@ const (
 	AppsPath = "/apps"
 
 	// OAuth2 sub-paths.
-	PathOAuth2         = "/oauth2"          // convention for Mattermost Apps, comes from OAuther
-	PathOAuth2Complete = "/oauth2/complete" // convention for Mattermost Apps, comes from OAuther
+	PathOAuth2             = "/oauth2"
+	PathMattermostRedirect = "/mattermost/redirect"
```

Comment:
do we need this const?

Reply:
I still need to sort out OAuth2 for Mattermost, and what callback (and redirect?) URLs should be reserved for those. The callback path is being registered and stored persistently in Mattermost when we create an OAuth2 Application for the App. The redirect const here is a reminder.